templates/blocks/social_media_cards.twig

```diff
@@ -16,31 +16,31 @@
 
 			<div class="container">
 				<div class="grid-row">
-					{% for key, single_image in images %}
+					{% for key, card in cards %}
 						<div class="grid-item">
-							<img src="{{ single_image.image_src }}" data-background-position="{{ single_image.focus_image }}"
-								srcset="{{ single_image.image_srcset }}" sizes="{{ single_image.image_sizes }}"
-								alt="{{ single_image.alt_text }}">
+							<img src="{{ card.image_src }}"
+								srcset="{{ card.image_srcset }}" sizes="{{ card.image_sizes }}"
+								alt="{{ card.alt_text }}">
 
 							<div class="share-strip">
-								<a href="https://www.facebook.com/share.php?u={{ single_image.social_url|url_encode|default(post_url) }}&quote={{ single_image.message|url_encode }}"
-								   title="{{ __('Share on Facebook') }}"
-									onclick="dataLayer.push({'event' : 'uaevent', 'eventCategory' : 'Social Share', 'eventAction': 'Facebook', 'eventLabel': '{{ single_image.social_url|default(post_url) }}'});"
+								<a href="https://www.facebook.com/share.php?u={{ card.social_url|default(post_url)|url_encode }}&quote={{ card.message|url_encode }}"
+								   title="{{ __('Share on Facebook', 'planet4-blocks-backend') }}"
```

Comment:
Here, the frontend text-domain(`planet4-blocks`) is applicable. The strings which display on frontend interface to user are mark as frontend text-domain.